src/base/bittorrent/torrenthandle.cpp

```diff
@@ -74,8 +74,7 @@ AddTorrentData::AddTorrentData()
     , hasSeedStatus(false)
     , skipChecking(false)
     , ratioLimit(TorrentHandle::USE_GLOBAL_RATIO)
-{
-}
+{}
```

Comment:
Please, either undo the changes like this one by hand or update your copy and run uncrustify ('{}' → '{\n}'). There was a mistake in the uncrustify config, sorry for that.